test/babashka/fs_test.clj

```diff
@@ -218,6 +218,15 @@
     (is (fs/exists? (fs/file tmp"foo2" "foo" "1"))
         "The nested destination directory is not created when it doesn't exist")))
 
+(deftest copy-tree-nested-ro-dir-test
+  (fs/with-temp-dir [tmp {}]
+    ;; https://github.com/babashka/fs/issues/122
+    (fs/create-dirs (fs/path tmp "src" "foo" "bar"))
+    (fs/set-posix-file-permissions (fs/path tmp "src" "foo") "r-xr-xr-x")
```

Comment:
This fails on Windows

Reply:
Looks like directories on Windows are always writable, according to [this]( https://answers.microsoft.com/en-us/windows/forum/all/all-folders-are-now-read-only-windows-10/0ca1880f-e997-46af-bd85-042a53fc078e) post.
I don't have a Windows system to check. I adjusted the test to skip this check on Windows.

Reply:
> I adjusted the test to skip this check on Windows.

Seems fine to me.